dangerfile.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * External dependencies
+ */
+
+import { danger, warn, markdown, results } from 'danger';
+
+// Skip danger check if "no ci" or "no danger" in latest commit
+const lastCommit = danger.git.commits.slice( -1 )[ 0 ].message;
+if (
+	lastCommit.includes( 'no ci' ) ||
+	lastCommit.includes( 'skip ci' ) ||
+	lastCommit.includes( 'no danger' ) ||
+	lastCommit.includes( 'skip danger' )
+) {
+	process.exit( 0 ); // eslint-disable-line no-process-exit
+}
+
+// No PR is too small to include a description of why you made a change
+if ( danger.github.pr.body.length < 10 ) {
+	warn( 'Please include a description of your PR changes.' );
+}
+
+// Use labels please!
+const ghLabels = danger.github.issue.labels;
+if ( ! ghLabels.find( l => l.name.toLowerCase().includes( '[status]' ) ) ) {
+	warn(
+		'The PR is missing at least one [Status] label. Suggestions: `[Status] In Progress`, `[Status] Needs Review`'
+	);
+}
+
+// Test instructions
+if ( ! danger.github.pr.body.includes( 'Testing instructions' ) ) {
+	warn( '"Testing instructions" are missing for this PR. Please add some' );
+}
+
+// skip if there are no warnings.
+if ( results.warnings.length > 0 || results.fails.length > 0 ) {
+	markdown(
+		"This is an automated check which relies on [`PULL_REQUEST_TEMPLATE`](https://github.com/Automattic/wp-calypso/blob/master/.github/PULL_REQUEST_TEMPLATE.md). We encourage you to follow that template as it helps Calypso maintainers do their job. If you think 'Testing instructions' are not needed for your PR - please explain why you think so. Thanks for cooperation :robot:"
+	);
+} else {
+	markdown( "That's a great PR description, thank you so much for your effort!" );
```

Comment:
If we don't emit any markdown, is the bot just quite? I don't know that we need the affirmation when folks get it right. But maybe we should try it for a bit and see how it feels.

Reply:
I'll remove it, let's see what happens. I'm fine with the bot being quiet if no warnings appear.

Reply:
@blowery I added the green flag again for the reasons mentioned above. Let me know if that works for you.